recipe/run_test.bat

```diff
@@ -1,8 +1,8 @@
-cd /D "%SRC_DIR%"
+cd /D nb_conda_kernels
```

Comment:
`SRC_DIR` is not longer allowed. I used `source_files` to copy the whole source (`nb_conda_kernels`) into the working directory. Not sure if this is correct though.